src/arrayMethodSort.js

```diff
@@ -4,9 +4,25 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length - 1; i++) {
+      if (
+        compareFunction(this[i], this[i + 1]) === true ||
+        compareFunction(this[i], this[i + 1]) > 0
+      ) {
+        const temporary = this[i];
+
+        this[i] = this[i + 1];
+
+        this[i + 1] = temporary;
+
+        i = -1;
+      }
+    }
+
+    return this;
```

Comment:
The method should return a sorted array, but the current implementation only performs one pass of bubble sort, which may not sort the array completely. You need to implement the full bubble sort algorithm or consider using a more efficient sorting algorithm.